services/static-webserver/client/source/class/osparc/dashboard/CardBase.js

```diff
@@ -112,6 +112,10 @@ qx.Class.define("osparc.dashboard.CardBase", {
           }
           return false;
         }
+        // if we get here, it means that it was shared-with-me via an organization
+        if (sharedWith === "shared-with-me") {
```

Comment:
shouldn't you use constants for these values?

Reply:
We could, it wouldn't fix the bug though

Reply:
it's more about maintainability. e.g. It's also easy to make a typo in the string while the linter can warn you if the constant is wrong

Reply:
I tend to like the constants approach too but I think we should perhaps have a story to clean it up throughout the  project and then keep it a standard practice. We used to call them magic strings ;)